api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb

```diff
@@ -484,7 +485,7 @@
    "id": "e9a80b25-c9a5-49e5-9609-a0f90ac2d107",
    "metadata": {},
    "source": [
-    "### Performing inference of cell class"
+    "### Performing inference of cell subclass"
```

Comment:
Is the meaning of "subclass" as opposed to "class/type" defined anywhere in the notebook?

Reply:
I added a description in top-level description. Thanks